src/dom/utils.ts

```diff
@@ -21,6 +21,10 @@ export function delay<T>(millis: number, value?: T): Promise<T> {
     return new Promise((resolve) => setTimeout(() => resolve(value), millis))
 }
 
+export function values<T>(m: SMap<T>): T[] {
```

Comment:
Object values?

Reply:
I must be stuck in the past

Reply:
also is there a better way to consume that spec repo? right I've just copied a build over but it's not great...